src/main/scala/org/broadinstitute/dsde/firecloud/webservice/MethodsApiService.scala

```diff
@@ -1,102 +1,120 @@
+                  perRequest(
+                    requestContext,
+                    Props(new AgoraPermissionActor(requestContext)),
+                    AgoraPermissionHandler.Post(url, fireCloudPermissions.map(_.toAgoraPermission)))
               }
-              perRequest(
-                requestContext,
-                Props(new AgoraPermissionActor(requestContext)),
-                AgoraPermissionHandler.MultiUpsert(agoraPermissions))
+            }
           }
         }
       }
     } ~
-    path( "configurations|methods".r / Segment / Segment / IntNumber / "permissions") { ( agora_ent, namespace, name, snapshotId) =>
-      val url = getUrlFromBasePath(agora_ent, namespace, name, snapshotId)
-      get { requestContext =>
-        // pass to AgoraPermissionHandler
-        perRequest(requestContext,
-          Props(new AgoraPermissionActor(requestContext)),
-          AgoraPermissionHandler.Get(url))
+    // routes that are only valid for methods
+    pathPrefix( "methods" ) {
+      val passthroughBase = remoteMethodsUrl
+      path( "definitions" ) {
+        get {
+          passthrough(s"$passthroughBase/definitions", HttpMethods.GET)
+        }
       } ~
-      post {
-        // explicitly pull in the json-extraction error handler from ModelJsonProtocol
-        handleRejections(entityExtractionRejectionHandler) {
-          // take the body of the HTTP POST and construct a FireCloudPermission from it
-          entity(as[List[FireCloudPermission]]) {
-            fireCloudPermissions => requestContext =>
-              perRequest(
-                requestContext,
-                Props(new AgoraPermissionActor(requestContext)),
-                AgoraPermissionHandler.Post(url, fireCloudPermissions.map(_.toAgoraPermission)))
-          }
-        } ~
-        delete {
-          complete(StatusCodes.MethodNotAllowed)
-        } ~
+      path( "permissions") {
         put {
-          complete(StatusCodes.MethodNotAllowed)
+          handleRejections(entityExtractionRejectionHandler) {
+            entity(as[List[MethodAclPair]]) { fireCloudPermissions =>
+              requestContext =>
+                val agoraPermissions = fireCloudPermissions map { fc =>
+                  EntityAccessControlAgora(Method(fc.method), fc.acls.map(_.toAgoraPermission))
+                }
+                perRequest(
+                  requestContext,
+                  Props(new AgoraPermissionActor(requestContext)),
+                  AgoraPermissionHandler.MultiUpsert(agoraPermissions))
+            }
+          }
         }
       } ~
-      delete {
-        complete(StatusCodes.MethodNotAllowed)
-      } ~
-      put {
-        complete(StatusCodes.MethodNotAllowed)
+      pathPrefix( Segment / Segment ) { (namespace, name) =>
+        path( "configurations" ) {
+          get {
+            passthrough(s"$passthroughBase/$namespace/$name/configurations", HttpMethods.GET)
+          }
+        } ~
```

Comment:
this route is the only new one added in this PR, to support broadinstitute/agora#205